pkg/windows/windows.go

```diff
@@ -258,6 +275,41 @@ func (vm *windows) EnsureRequiredServicesStopped() error {
 	return nil
 }
 
+// removeServices removes all services installed by WMCO
```

Comment:
`ensures all services installed by WMCO are removed`

Might want to rename this `ensureServicesAreRemoved()`